src/hip/gpu_getxc.cu

```diff
@@ -12,10 +12,10 @@ __constant__ gpu_simulation_type devSim_dft;
 static float totTime;
 #endif
 
-#include "gpu_getxc.h"
```

Comment:
What is the reason for renaming the file?

Reply:
According to my tests, functional specific global kernels (for XC potential and XC gradient) are significantly faster than generic kernels (i.e. single global kernel that can call many functional device kernels) on AMD cards. I was planning to have preprocessable version of gpu_getxc.h and renamed the existing version to gpu_getxc_generic.h.

Reply:
Got it. Thanks.